Add extra turb length/time calculation

clarkpede opened the pull request:
Currently, length and timescales for the v2f method are only calculated
when the source residual is calculated. This leads to problems where the
eddy viscosity is needed but the residual is not.  For example, the eddy
viscosity is calculated when a restart file is loaded, which means that
incorrect values of T and L are used.

This commit adds an extra computation of the turb time and lengthscales,
to ensure that they are properly initialized when the eddy viscosity is
calculated.

clarkpede committed:
Add extra turb length/time calculation

Currently, length and timescales for the v2f method are only calculated
when the source residual is calculated. This leads to problems where the
eddy viscosity is needed but the residual is not.  For example, the eddy
viscosity is calculated when a restart file is loaded, which means that
incorrect values of T and L are used.

This commit adds an extra computation of the turb time and lengthscales,
to ensure that they are properly initialized when the eddy viscosity is
calculated.

trevilo commented:
Looks like this is enough to change the expected results on the v2-f flat plate BL test.  So, I'd update those.  Otherwise, looks great to me.  Go ahead and merge whenever you're ready.

clarkpede committed:
Adjust test values for v2f flatplate test

Because of an improvement in the v2f calculations, the test values
need updating.
